tests/integration/test_client.py

```diff
@@ -204,22 +160,27 @@ def _get_tables(core):
         index += 1
 
 
-async def test_exists(client: Client):
-    name = str(uuid.uuid4())
+async def test_exists(client: Client, table_name_prefix: str):
+    name = table_name_prefix + str(uuid.uuid4())
```

Comment:
maybe a bit OT... what do you think about `secrets.token_hex(4)` instead of UUID?

Reply:
either is probably totally fine.